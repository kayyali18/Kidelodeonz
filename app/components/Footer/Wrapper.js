import styled from 'styled-components'

const Wrapper = styled.footer`
  display: flex;
  align-items: center;
  justify-content: space-around;
  background-color: #402c28;
`

export default Wrapper
