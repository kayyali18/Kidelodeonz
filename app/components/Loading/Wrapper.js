import styled from 'styled-components'

const DotWrapper = styled.div`
  display: flex;
  align-items: flex-end;
`

export default DotWrapper
