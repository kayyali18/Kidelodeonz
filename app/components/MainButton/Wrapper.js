import { Link } from 'react-router-dom'
import styled from 'styled-components'

const Wrapper = styled(Link)`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  height: 100%;
  text-align: center;
  position: relative;
  text-decoration: none;
`

export default Wrapper
