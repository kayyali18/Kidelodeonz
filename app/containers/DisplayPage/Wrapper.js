import styled from 'styled-components'

const Wrapper = styled.section`
  display: grid;
  grid-template-rows: 1fr 2fr
  height: 100vh;
  width: 80vw;
  margin: auto;
`
export default Wrapper
