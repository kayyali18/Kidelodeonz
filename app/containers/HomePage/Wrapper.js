import styled from 'styled-components'

const Wrapper = styled.section`
  display: grid;
  grid-template-rows: repeat(auto-fit, minmax(1fr, 1fr));
  grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
  gap: calc(10px + 26 * ((100vw - 320px) / 680));
  height: 100%;
  width: 80vw;
  margin: auto;
`
export default Wrapper
