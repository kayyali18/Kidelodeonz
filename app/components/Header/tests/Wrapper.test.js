import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Wrapper from '../Wrapper'

describe('<Wrapper />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(<Wrapper />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should be a <Nav />', () => {
    const renderedComponent = renderer.create(<Wrapper />).toJSON()
    expect(renderedComponent.type).toEqual('nav')
  })
})
