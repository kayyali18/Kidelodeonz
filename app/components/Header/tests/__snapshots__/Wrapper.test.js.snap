// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  background-color: #fff;
  height: 100%;
  position: -webkit-sticky;
  position: sticky;
  top: 0;
}

<nav
  className="c0"
/>
`;
